docs/configuration.md

```diff
@@ -56,7 +56,7 @@ To do that, use the same format as with "Accepts multiple values" above:
 ```ini
 greetingMessage[] = Welcome to the server!
 greetingMessage[] = This is the second line of the greeting.
-greetingMessage[] = 
+greetingMessage[] =
```

Comment:
Unnecessary formatting change

Reply:
Fixed -- sorry, my editor,  Atom doesn't like hanging spaces. Added back in to revert the formatting change